config/locales/status.en.yml

```diff
@@ -1,5 +1,10 @@
 en:
   status:
+    label:
```

Comment:
@thomasleese Wasn't sure where to put these translations. Started out with `I18n.t("status.consent.label.status")` but then thought it didn't read well, so went with `I18n.t("status.label.consent")`. Happy to take other suggestions.

Reply:
Yeah, I think that works well. By having the dynamic part at the end it also allows us to do things like `I18n.t("consent", scope: %i[status label])` which I think is nice.